pinot-broker/src/main/java/org/apache/pinot/broker/api/ManualAuthorization.java

```diff
@@ -0,0 +1,36 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.pinot.broker.api;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+
+/**
+ * Annotation to be used on top of REST endpoints. Methods annotated with this annotation don't perform default
+ * authorization via AuthenticationFilter. This is useful when performing authorization manually via calls to
+ * {@code AuthenticationFiler.validatePermissions()}
+ */
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.METHOD)
+public @interface ManualAuthorization {
+}
```

Comment:
This is already defined in controller module. Can you move it to a common place, so it's not duplicated in two places? Maybe in `pinot-common > src > main > java > org > apache > pinot > common > auth`?

Reply:
I tried that in another attempt, such change will trigger a huge refactor.
Am I allowed to make breaking changes like removing public interfaces in controller module? @apucher

Reply:
This is not an interface change. It's just moving the same interface to another module to prevent duplicate code.
I assume the huge refactoring you're referring to is for just package name change, right? If that's the only change, we should go for it.

Reply:
the `@ManualAuthorization` annotation was just introduced recently. It should be fine to move it around.

In terms of preventing a `huge refactor` - what issues are you running into there?

Reply:
I was thinking that I have to keep those interfaces in pinot-controller. If it is fine to relocate them into different packages (downstream may fail to build), refactor will be much smaller.
Let me come up with something

Reply:
Thanks, all fixed now. Sorry for the slow update but it should be working now.